examples/arm/run.sh

```diff
@@ -118,8 +118,7 @@ function build_quantization_aot_lib()
         -Bcmake-out-aot-lib \
         "${et_root_dir}"
 
-    n=$(nproc)
-    cmake --build cmake-out-aot-lib -j"$((n - 5))" -- quantized_ops_aot_lib
+    cmake --build cmake-out-aot-lib --parallel -- quantized_ops_aot_lib
```

Comment:
Ok for now, if i were to guess `--parallel` or `-j` might push oom? `n - 5` may be too conservative. Let's see how this fare.

Reply:
My thinking also, lets see if its a problem